constants/pathTranslations.ts

```diff
@@ -0,0 +1,22 @@
+const queryParamsMap = {
```

Comment:
Może użyć `enum`a?

Reply:
Nie można, bo enum nie ma odpowiednika w runtime.

Reply:
Hmm, ale TypeScript przecież [zamienia `enum`y na obiekty do runtime'u](https://www.typescriptlang.org/docs/handbook/enums.html#enums-at-runtime).

Reply:
Jeżeli chcesz trzymać QueryParams i SortOptions osobno to i tak przerób sobie to na dwa enumy
```ts
export enum PageQueryParams {
  Sort = 'sortowanie',
  Page =  'strona',
}
```
i
```ts
export enum SortOptions  {
  Votes = 'najwięcej-głosów',
  Answers = 'najwięcej-odpowiedzi',
  Views = 'najwięcej-odwiedzin',
  Hot = 'gorące',
  Date =  'najnowsze',
}
```

Reply:
Chciałem tutaj zasugerować enumy, ale konwencja wymaga żeby wartości w enumie były z wielkiej litery. Obiekt faktycznie lepiej się do tego nada